changelog.txt

```diff
@@ -1,6 +1,10 @@
+1.1.3-preview
+=============
+This release contains updates to Xamarin.Android.Support v27.0.2 and MonoAndroid8.1 (#553 #520).

```

Comment:
do we want to explain why this is needed? tooling (VS 2017) update?

Reply:
This was a customer ask...lots of customers asking that we move to v27.0.2 and only MonoAndroid8.1 is supported w/that version. should I explain more here?

---
In reply to: [190842857](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/563#discussion_r190842857) [](ancestors = 190842857)

Reply:
Great :)